src/styles/header.scss

```diff
@@ -0,0 +1,60 @@
+.header {
+  width: 100%;
+  padding: 0 50px;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), the `padding` property should be written in the shorthand form, if possible. So, you should write `padding: 0 50px;` as `padding: 0 50px;`